Demo/Tests/ErrorTests.swift

```diff
@@ -52,6 +52,32 @@ class ErrorTests: QuickSpec {
             }
         }
 
+        describe("mapping a result with empty data") {
+            let response = Response(statusCode: 200, data: NSData())
+
+            it("fails on mapJSON with default parameter") {
+                var succeeded = false
+                do {
+                    let _ = try response.mapJSON()
+                } catch {
+                    succeeded = true
```

Comment:
The wording here confused me initially (using `succeeded` to indicate that an error was successfully caught). What do you think about changing this variable to be something like `mapJSONFailed` or similar?